tests/admin_inlines/tests.py

```diff
@@ -1885,17 +1883,15 @@ def test_added_stacked_inline_with_collapsed_fields(self):
         )
         self.selenium.find_element(By.LINK_TEXT, "Add another Child").click()
         test_fields = ["#id_child_set-0-name", "#id_child_set-1-name"]
-        show_links = self.selenium.find_elements(By.LINK_TEXT, "SHOW")
-        self.assertEqual(len(show_links), 2)
+        summaries = self.selenium.find_elements(By.TAG_NAME, "summary")
+        self.assertEqual(len(summaries), 3)
```

Comment:
I increased the count from 2 to 3 here on purpose.

There are two visible fieldsets, but a third one is hidden in the code that is used to copy a new fieldset when clicking "Add another Child".

You can see that `test_collapsed_inlines()` already looked for three elements.